plugin-server/src/main/ingestion-queues/jobs-buffer-consumer.ts

```diff
@@ -0,0 +1,90 @@
+import { StatsD } from 'hot-shots'
+import { EachBatchHandler, Kafka } from 'kafkajs'
+import { KafkaProducerWrapper } from 'utils/db/kafka-producer-wrapper'
+
+import { KAFKA_EVENTS_DEAD_LETTER_QUEUE, KAFKA_JOBS } from '../../config/kafka-topics'
+import { EnqueuedPluginJob, JobName } from '../../types'
+import { status } from '../../utils/status'
+import { GraphileWorker } from '../graphile-worker/graphile-worker'
+import { instrumentEachBatch, setupEventHandlers } from './kafka-queue'
+
+export const startJobsBufferConsumer = async ({
```

Comment:
This file seems like it's mostly a copy of the buffer consumer. 

I suspect your opinion is that reusing the code with some topic changes would be more confusing? Perhaps because inferring capabilities might be harder at a glance?

I'm happy with this, so the comment is non-blocking, but I think I'd reuse the code personally. Make a general `jobsConsumer` and change the topic based on capabilities.

Reply:
Also another nit is let's maybe avoid using the term 'buffer' here given that might lead to confusion

Reply:
> This file seems like it's mostly a copy of the buffer consumer.

I'm going with the [AHA vs DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) methodology here, it might be worth abstracting if we end up getting bugs that we need to fix in multiple places, but atm it feel unnecessary.